app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <title><%= content_for?(:title) ? yield(:title) : SiteSetting.title %></title>
     <meta name="description" content="<%= @description_meta || SiteSetting.site_description %>">
-    <meta name="discourse_theme_ids" content="<%= theme_ids&.join(",") %>">
+    <meta name="discourse_theme_ids" content="<%= theme_id %>">
```

Comment:
Should we update the `name` of this meta tag? (and all the JS which references it?)

Reply:
Good catch :+1: